app/i18n/en-us/install.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+return array(
+	'action' => array(
+		'finish' => 'Complete installation',
+		'fix_errors_before' => 'Please fix errors before continuing to the next step.',
```

Comment:
```suggestion
		'fix_errors_before' => 'Please fix any errors before continuing to the next step.',
```

Reply:
'any' does sound better, but sounds to imply there 'might' be errors as opposed to there 'are' errors.  This sounds like it would be used in context there are errors that must be fixed.  Will consider revising

Reply:
Please fix the errors…?

Reply:
I'm going with 'Please fix all errors...'